en/flight_controller/crazyflie2.md

```diff
@@ -117,10 +117,9 @@ obtain the latest QGC source code (master) and build it.)
 
 ## Hardware Setup
 
-Uptill now we have been able to fly crazyflie with precise control in Stabilised and Altitude modes. 
+Crazyflie is able to fly with precise control in Stabilised, Altitude and Position modes. 
```

Comment:
What relevance has this sentence got to Hardware Setup section?

Reply:
What does precise control mean?

Reply:
I would remove "with precise control" and also I think stabilized has a 'z' and not an 's'.

Reply:
> What relevance has this sentence got to Hardware Setup section?

I wanted to link the hardware needed for each of these modes in the next few lines.

I can remove "precise control" (It was in the old crazyflie document not written by me).

Reply:
> I think stabilized has a 'z' and not an 's'.

Yes, the mode is "Stabilized"

In the general case, docs are an odd mix of US and "real" English :-). I don't enforce one or the other when subediting, but I do make sure that within a document the usage is consistent.